### PR TITLE
Merge storefront-branch into develop

### PR DESCRIPTION
**Pull request by vnys**
This pr merges the storefront branch into the develop branch.

### CODE REVIEW

File: apps/storefront/src/components/ComponentStatus/ComponentStatus.jsx [hunk 1]
```diff
@@ -33,27 +34,46 @@ const ComponentStatus = () => {
...
-      <thead>
-        <Tr>
-          {headers.map((text, index) => (
-            <ThH key={camelify(text)} scope="col">
+      <Head>
+        <Row>
+          {headers.map((text) => (
+            <Cell
+              as="th"
+              style={{ textAlign: 'left' }}
```
Comment (mimarz) [inline apps/storefront/src/components/ComponentStatus/ComponentStatus.jsx:46]: We fixed this so should work without this prop :D

File: apps/storefront/src/components/ComponentStatus/ComponentStatus.jsx [hunk 2]
```diff
@@ -33,27 +34,46 @@ const ComponentStatus = () => {
...
-          <Tr key={component.component + index}>
-            <ThV key={component.component + index} scope="row">
-              <Link to={`/components/${kebabify(component.component)}`}>
+          <Row key={component.component + index}>
+            <Cell
+              as="th"
+              key={component.component + index}
+              scope="row"
+              style={{
+                textAlign: 'left',
```
Comment (mimarz) [inline apps/storefront/src/components/ComponentStatus/ComponentStatus.jsx:63]: here to

File: apps/storefront/src/components/Titles.jsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+const tocAdjustment = css`
+  display: block;
+  content: ' ';
+  margin-top: -80px;
+  height: 80px;
+  visibility: hidden;
+`
+
+const StyledH1 = styled(Typography)`
```
Comment (mimarz) [inline apps/storefront/src/components/Titles.jsx:15]: Should these changes be added to the `<Typography>` component? 🤔

File: apps/storefront/src/components/ComponentStatus/ComponentStatus.jsx (hunk 2 above)
Comment (vnys) [reply apps/storefront/src/components/ComponentStatus/ComponentStatus.jsx:63]: Actually, it seems this was solved by setting `text-align: left` on `thead` – so the vertical th’s are still centered.

File: apps/storefront/src/components/Titles.jsx (hunk 3 above)
Comment (vnys) [reply apps/storefront/src/components/Titles.jsx:15]: Maybe… 💭

File: apps/storefront/src/components/ComponentStatus/ComponentStatus.jsx (hunk 1 above)
Comment (vnys) [reply apps/storefront/src/components/ComponentStatus/ComponentStatus.jsx:46]: Removed in #455

File: apps/storefront/src/components/ComponentStatus/ComponentStatus.jsx (hunk 2 above)
Comment (vnys) [reply apps/storefront/src/components/ComponentStatus/ComponentStatus.jsx:63]: Fixed in #454

**Pull request merged**